cmd/src/search_jobs_results.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+)
+
+func init() {
+	usage := `retrieves the results of a search job in JSON Lines (jsonl) format.
+Examples:
+
+	Get the results of a search job:
+	  $ src search-jobs results -id 999
```

Comment:
This comment applies to most of the other sub-commands as well. 

Since we always expect users to specify an id, we could just make it a positional argument, which also makes it obvious that it isn't optional.